tests/system/robot/chromeTests.py

```diff
@@ -649,3 +649,89 @@ def test_i10840():
 		nextActualSpeech,
 		"column 2  items"
 	)
+
+
+def test_preventDuplicateSpeechFromDescription_browse_downArrow():

```

Comment:
NVDA has never reported description for title when reading in browse mode with up and down arrow, whether it is duplicate or not. Thus this test might be a bit misleading. I'd personally remove it. But if you don't want to do this, then perhaps add another test before it that tests that a non-duplicate title is also not read.